iron-swipeable-container.html

```diff
@@ -0,0 +1,266 @@
+a curved or horizontal transition, but the transition duration and properties
+can be customized.
+
+Example:
+
+    <iron-swipeable-container>
+      <div>I can be swiped</div>
+      <paper-card heading="Me too!"></paper-card>
+    </iron-swipeable-container>
+
+To disable swiping on individual children, you must give them the `.disable-swipe`
+class. Alternatively, to disable swiping on the whole container, you can use its
+`disable-swipe` attribute:
+
+    <iron-swipeable-container>
+      <div class="disable-swipe">I cannot be swiped be swiped</div>
+      <paper-card heading="But I can!"></paper-card>
+    </iron-swipeable-container>
+
+    <iron-swipeable-container disable-swipe>
+      <div>I cannot be swiped</div>
+      <paper-card heading="Me neither :("></paper-card>
+    </iron-swipeable-container>
+
+It is a good idea to disable text selection on any of the children that you
+want to be swiped:
+
+    .swipe {
+      -moz-user-select: none;
+      -ms-user-select: none;
+      -webkit-user-select: none;
+      user-select: none;
+      cursor: default;
+    }
+
+@group Iron Elements
+@element iron-swipeable-container
+@demo demo/index.html
+-->
+
+<dom-module id="iron-swipeable-container">
+  <template>
+    <content id="content"></content>
+  </template>
+</dom-module>
+
+<script>
+
+  Polymer({
+    is: 'iron-swipeable-container',
+
+    /**
+     * Fired when a child element is swiped away.
+     *
+     * @event iron-swipe-away
+     */
+
+    properties: {
+      /**
+       * The style in which to swipe the card. Currently supported
+       * options are `curve | horizontal`. If left unspecified, the default
+       * is assumed to be `curve`.
+       */
+      swipeStyle: {
+       type: String,
+       value: 'curve'
+      },
+
+      /**
+       * If true, then the container will not allow swiping.
+       */
+      disableSwipe: {
+        type: Boolean,
+        value: false
+      },
+
+      /**
+       * The ratio of the width of the element that the translation animation
+       * should happen over. For example, if the `widthRatio` is 3, the
+       * animation will take place on a distance 3 times the width of then
+       * element being swiped.
+       */
+      widthRatio: {
+       type: Number,
+       value: 3
+      },
+
+      /**
+       * The ratio of the total animation distance after which the opacity
+       * transformation begins. For example, if the `widthRatio` is 1 and
+       * the `opacityRate` is 0.5, then the element needs to travel half its
+       * width before its opacity starts decreasing.
+       */
+      opacityRate: {
+       type: Number,
+       value: 0.2
+      },
+
+      /**
+       * The CSS transition applied while swiping.
+       */
+      transition: {
+        type: String,
+        value: '300ms cubic-bezier(0.4, 0.0, 0.2, 1)'
+      },
+
+      /**
+       * The CSS transition property applied while swiping.
+       */
+      transitionProperty: {
```

Comment:
I think I'm just going to make it private. I think transform and opacity is the only thing we should support.